core/src/avm2/globals/flash/media/sound.rs

```diff
@@ -264,14 +264,28 @@ pub fn load<'gc>(
 /// `Sound.loadCompressedDataFromByteArray`
 pub fn load_compressed_data_from_byte_array<'gc>(
     activation: &mut Activation<'_, 'gc>,
-    _this: Object<'gc>,
-    _args: &[Value<'gc>],
+    this: Object<'gc>,
+    args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    avm2_stub_method!(
-        activation,
-        "flash.media.Sound",
-        "loadCompressedDataFromByteArray"
-    );
+    let bytearray = args.get_object(activation, 0, "bytes")?;
+    let bytes_length = args.get_u32(activation, 1)?;
+    let bytearray = bytearray
+        .as_bytearray_mut(activation.context.gc_context)
```

Comment:
I think you can just use `as_bytearray` here, as read_bytes does not need mutability. (Position is stored in a Cell)

Reply:
Good catch - I've changed it